pystdf/__main__.py

```diff
@@ -0,0 +1,55 @@
+
+"""
+Execute the pystdf module: apply a conversion to an STDF file.
+
+Usage:
+    python -m pystdf [conversion] [stdf-file]
+
+    Conversion is either txt, xml, xlsx, slice or count.
+"""
+
+import sys
+
+
+def print_help():
+    print("""pystdf
+    
+Usage:
+    python -m pystdf [conversion] [stdf-file]
+    
+    Conversion is either txt, xml, xlsx, slice or count.
+""")
+
+
+def main():
+    if len(sys.argv) < 3:
+        print_help()
+        return
+        
+    conversion, file = sys.argv[1:3]
+    args = sys.argv[3:]
+        
+    if conversion not in ['txt', 'xml', 'xlsx', 'slice', 'count']:
```

Comment:
Some of these seem like format conversions (txt, xml, xlsx), slicing seems like an output option, and count seems like a summary function. Might be worth splitting these into different subcommands and option arguments.